modules/streaming/src/main/scala/io/circe/streaming/ParsingConfiguration.scala

```diff
@@ -0,0 +1,11 @@
+package io.circe.streaming
+
+import jawn.AsyncParser
+
+case class ParsingConfiguration(parseMode: AsyncParser.Mode)
+
+object ParsingConfiguration {
+
+  implicit val defaultParsingConfiguration: ParsingConfiguration =
```

Comment:
I don't think we have to rename. We can keep `stringParser`/`byteParser` in place and just document what parsing mode they use.

I guess I'm missing something, but I'm not sure I understand the reason we need `SingleValue` given `StreamValue` exists.